src/app/containers/ATIAnalytics/index.jsx

```diff
@@ -5,24 +5,27 @@ import CanonicalATIAnalytics from './canonical';
 import AmpATIAnalytics from './amp';
 import ArticleAtiParams from './params/article';
 import FrontPageAtiParams from './params/frontpage';
+import MediaPageAtiParams from './params/media';
 import { articleDataPropTypes } from '../../models/propTypes/article';
 import { frontPageDataPropTypes } from '../../models/propTypes/frontPage';
 
 const ATIAnalytics = ({ data }) => {
   const { pageType, platform } = React.useContext(RequestContext);
 
-  let pageviewParams = '';
-  switch (pageType) {
-    case 'article':
-      pageviewParams = ArticleAtiParams(data);
-      break;
-    case 'frontPage':
-      pageviewParams = FrontPageAtiParams(data);
-      break;
-    default:
-      return null;
+  // The functions that converts pageData to ATI params
+  const pageTypeHandlers = {
+    article: ArticleAtiParams,
```

Comment:
I also think the comment above wouldn't be needed if the functions were named better e.g. `getArticleAtiParams`. The capitalisation also suggests it's a React component

Reply:
@jroebu14 updated

This actually is a react component.  Confused me at first as it just returns a string, but it needs to be a component in order to use `React.context`